src/components/LineContent/index.tsx

```diff
@@ -53,11 +52,14 @@ export default class LineContent extends Component<any, any> {
         }
 
         return (
-            <span className={lineContent} style={style}>
-                {data &&
-                    data.map((part, n) => (
-                        <LinePart part={part} format={formatPart} key={`line-${number}-${n}`} />
-                    ))}
+            <span
+                className={lineContent}
+                style={{ width: "100%", display: "inline-block" }}
```

Comment:
Why are we hardcoding the style instead of passing it in as a prop?

Reply:
I don't remember but I'll put it back 😅